modules/swagger-codegen/src/main/resources/JavaSpringMVC/apiException.mustache

```diff
@@ -1,10 +1,71 @@
 package {{apiPackage}};
 
+import java.util.Map;
+import java.util.List;
+
 {{>generatedAnnotation}}
-public class ApiException extends Exception{
-	private int code;
-	public ApiException (int code, String msg) {
-		super(msg);
-		this.code = code;
-	}
+public class ApiException extends Exception {
```

Comment:
Ok morgen ff

Christiaan Postma <notifications@github.com> schreef op 19 april 2016 18:15:42 CEST:
>>  {{>generatedAnnotation}}
>> -public class ApiException extends Exception{
>> -	private int code;
>> -	public ApiException (int code, String msg) {
>> -		super(msg);
>> -		this.code = code;
>> -	}
>> +public class ApiException extends Exception {
>
>The calling program still needs to "know" too much about the protocol
>that's being used. Let's discuss.
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/VisionsConnectedEurope/swagger-codegen/pull/2/files/1e53f5bbe27e727be653ba8c30341ea7b68cffd2#r60262024

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.